Client/Frontend/Library/BookmarksPanel.swift

```diff
@@ -18,6 +18,14 @@ private struct BookmarksPanelUX {
     static let RowFlashDelay: TimeInterval = 0.4
 }
 
+private var disclosureIndicator: UIImageView {
+    let disclosureIndicator = UIImageView()
+    disclosureIndicator.image = UIImage(named: "menu-Disclosure")?.withRenderingMode(.alwaysTemplate)
+    disclosureIndicator.tintColor = UIColor.theme.tableView.accessoryViewTint
```

Comment:
looks great 👍 I am just wondering that the system disclosure indicator can't be tinted easily 🤔

Reply:
@ecotopian It has always been the case for TableViews, hasn't it?
I copied the `disclosureIndicator` implementation from Ecosia Settings, wish I think you added yourself.

Reply:
then it must be the right way 😆